references/regex.json

```
@@ -1 +1,5 @@
-{}
+{
+    "submitter_id": "^[A-Za-z0-9\\-\\._]{1,64}$",
```

Do we really need to restrict submitters to a max length 64 restriction? I think our only rule is the need for min length of 1, not sure we care about max length at all.

```suggestion
    "submitter_id": "^[A-Za-z0-9\\-\\._]+$",
```